wireformat/metrics/metrics.go

```diff
@@ -37,18 +37,19 @@ type Response struct {
 	NewGracePeriod time.Duration        `json:"new-grace-period"`
 }
 
+// EnvironmentResponses is a map of model UUID to wireformat responses.
```

Comment:
Will consider it for a followup. If we can coordinate this with a /v3/metrics endpoint, we can even rename the wireformat fields.